settings.gradle.kts

```diff
@@ -189,6 +189,7 @@ module("ide") {
     subproject("ide-plugins")
     subproject("problems")
     subproject("problems-api")
+    subproject("problems-rendering")
```

Comment:
same as above

Reply:
Answered above, follow up in the main thread